src/subscribables/dependentObservable.js

```diff
@@ -57,6 +57,34 @@ ko.computed = ko.dependentObservable = function (evaluatorFunctionOrOptions, eva
         isSleeping = false;
     }
 
+    function subscribeToDependency(target) {
```

Comment:
This function will be quite a lot more expensive than the usual 'subscribe' if we go into the 'deferred' code path, both in memory and CPU use, as now it constructs several extra function instances with their own closures. This once for each dependency, every time any 'deferred' computed is re-evaluated.

It might be OK, but is there any possibility of dropping these dynamically created closures in favour of static function instances and variables that get mutated?